db/schema.rb

```diff
@@ -297,10 +298,10 @@
     t.string "last_sign_in_ip"
     t.integer "role"
     t.string "old_user_id"
+    t.string "phone"
```

Comment:
Don't think these two (phone lines) should be here. Can you revert them?

Reply:
they're in the schema on main https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/blob/main/db/schema.rb

Reply:
Yep, I don't mean they should be removed altogether. Just that this PR shouldn't be changing them (since there's no migration for that here).

Reply:
Fixed it 🙂